doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,140 @@
+.. _tutorial_qemu:
+
+Qemu
+****
+
+In this tutorial, we will create our first cloud-init user data config and
+deploy it into a Qemu_ virtual machine. We'll be using Qemu for this tutorial
+which is a popular emulator for running virtual machines on Linux. Several
+popular virtual machine tools use Qemu, including Libvirt, LXD, and Vagrant.
+
+Install Qemu
+============
+
+.. code-block:: sh
+
+    $ sudo apt install qemu-system-x86
```

Comment:
Is it conventional to use the $ signs in the code snippets? A couple of times they caused me problems when I was copy/pasting them.

Reply:
Yes, this is a convention. It helps to disambiguate whether a command is single-line or multi-line, and is sometimes used to indicate non-root vs root commands (using $ and #, respectively).

Consider the heredoc commands below (line 41). The $ here helps users to understand that these are not individual commands to be executed in sequence, but are part of a multi-line shell command.

I've seen this question come up enough times in IRC/Stack Exchange/etc that I agree it is potentially worth dropping to avoid confusion. If not for the heredoc commands (see line 41), I think this would be easy to eliminate them all.

My concern is that if I saw the following while skimming a *nix tutorial, I would have a moment of pause because lines after the first one are not valid shell commands. 
```
    cat << EOF > user-data
    #cloud-config
    password: passw0rd
    chpasswd:
      expire: False
    EOF
```
Is this more readable to others? If we're alright with this format I'd be fine with dropping all of the $s.

Thoughts?

Reply:
It does also indicate what lines are command and what are output:

```
$ pwd
/home/aciba
```

And prevents from disaster while copying and pasting commands, if I copy-paste `$ rm /` and press enter, then I will get an error because of the invalid `$`.